robottelo/host_helpers/__init__.py

```diff
@@ -8,5 +9,9 @@ class ContentHostMixins(SystemFacts, VersionedContent):
     pass
 
 
-class SatelliteMixins(ContentInfo, Factories):
+class CapsuleMixins(CapsuleInfo, ContentHostMixins):
+    pass
+
+
+class SatelliteMixins(ContentInfo, Factories, CapsuleMixins):
```

Comment:
Same thing here about not needing capsule mixins

Reply:
CapsuleMixins are needed,  so without that it is not possible to run Capsule methods in CapsuleInfo via Satellite. 
`self._satellite.get_available_capsule_port()` is no longer reachable, or at least the way it is written right now

Reply:
CapsuleMixins moved under Capsule, thanks Jake